intl/messages/pt.json

```diff
@@ -91,6 +91,11 @@
     ]
   },
 
+  "serviceWorker": {
+    "sectionTitle": "PT-Service Worker",
+    "sectionDesc": "PT-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris tristique quis dui nec pellentesque. Cras sed porta neque, in vulputate turpis. Maecenas convallis quam urna, eu volutpat enim rhoncus ut."
```

Comment:
It is missing the text that goes next to the toggle.

Reply:
Done.